admin/code/LeftAndMain.php

```diff
@@ -451,7 +451,7 @@ public function init() {
 			]
 		]);
 
-		Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/javascript/dist/bundle-leftandmain.js', [
+		Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/javascript/dist/bundle-legacy.js', [
```

Comment:
Because it is. It contains all the entwine code, which is there for legacy reasons (we don't have time to recreate it all in React just yet), and shouldn't be modified without being aware of that fact. I'm not too fussed about "legacy" terminology, but `leftandmain` is wrong - other sections extend of the `LeftAndMain` PHP class (e.g. `CampaignAdmin`), and aren't powered by code in `bundle-leftandmain.js`